src/sentry/static/sentry/app/views/settings/organizationTeams/index.jsx

```diff
@@ -1,103 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import Reflux from 'reflux';
-import createReactClass from 'create-react-class';
-
-import {loadStats} from 'app/actionCreators/projects';
-import {sortArray} from 'app/utils';
-import ProjectsStore from 'app/stores/projectsStore';
-import SentryTypes from 'app/sentryTypes';
-import TeamStore from 'app/stores/teamStore';
-import withApi from 'app/utils/withApi';
-import withOrganization from 'app/utils/withOrganization';
-
-import OrganizationTeams from './organizationTeams';
-
-const OrganizationTeamsContainer = createReactClass({
-  displayName: 'OrganizationTeamsContainer',
-
-  propTypes: {
-    api: PropTypes.object,
-    organization: SentryTypes.Organization,
-    routes: PropTypes.arrayOf(PropTypes.object),
-  },
-
-  mixins: [
-    Reflux.listenTo(TeamStore, 'onTeamListChange'),
-    Reflux.listenTo(ProjectsStore, 'onProjectListChange'),
-  ],
-
-  getInitialState() {
-    return {
-      teamList: sortArray(TeamStore.getAll(), function(o) {
-        return o && o.name;
-      }),
-      projectList: sortArray(ProjectsStore.getAll(), function(o) {
-        return o && o.name;
-      }),
-      projectStats: {},
-    };
-  },
-
-  componentDidMount() {
-    this.fetchStats();
-  },
-
-  fetchStats() {
-    loadStats(this.props.api, {
-      orgId: this.props.params.orgId,
-      query: {
-        since: new Date().getTime() / 1000 - 3600 * 24,
-        stat: 'generated',
-        group: 'project',
-      },
-    });
```

Comment:
It's possible this loads data as a side effect for some other component, but because this component has state property of `projectsStats` and listened to the store, I assume it was intended for this component.